piksi_tools/console/console.py

```diff
@@ -223,13 +235,13 @@ class SwiftConsole(HasTraits):
 
```

Comment:
We need different callbacks for deprecated msg_prints vs msg_logs.  the msg_prints will have log level inferred, the msg_logs will go directly into the table with the level that comes in.